src/src/js/grids/vuetify/dataAdaptor.ts

```diff
@@ -21,6 +23,10 @@ export class DataAdaptor extends ObApiClient {
         filters.push(`limit=${query.limit}`);
         filters.push(this.constructSortQuery(query.orderBy));
 
+        if (this._parentId) {
```

Comment:
In the syncfusion front-end, the `executeApi` is more complex and with more parameters. Why can't we do this here? Why can't `parentId` be an optional parameter to pass to `executeApi` instead of being passed in the ctor?

![image](https://github.com/onebeyond/onebeyond-studio-obelisk-fe-vuetify/assets/21156129/9c4fdbe3-b516-42d7-bf26-8cadd11471ce)